Lib/defcon/objects/layer.py

```diff
@@ -223,14 +224,13 @@ def insertGlyph(self, glyph, name=None):
         # it is crucially important that the data be reconstructed
         # in its entirety so that the parent data is properly set
         # in all of the various objects.
-        from copy import deepcopy
+        # FIXME: Please Explain!!
```

Comment:
Apparently I left that comment there. I think it was as a request to make that "crucially important ..." comment more self contained. I.e.  Append a line that explains why it is so important, like "Because we don't want to have a glyph that is referenced in two distinct layers at the same time."

Also, I think I'm personally not sure if I like the fact the `insertGlyph` in fact creates a copy and inserts that. Probably, when writing the deserialization stuff I was thinking that recreating the glyph object is unneeded and hence too expensive, because I would insert a newly made Glyph not referenced anywhere else. But, I must admit that I'm not currently into this topic and that it's hard to give a better considered reply. Hence I'm staying a bit general.